server/member.go

```diff
@@ -108,6 +108,15 @@ func (s *Server) registerMemberRoutes(g *echo.Group) {
 		if err := jsonapi.UnmarshalPayload(c.Request().Body, memberPatch); err != nil {
 			return echo.NewHTTPError(http.StatusBadRequest, "Malformed patch member request").SetInternal(err)
 		}
+		// When archiving a owner, make sure he's not the only unarchived owner
+		if member.Role == "OWNER" && *memberPatch.RowStatus == "ARCHIVED" {
+			countResult, err := s.store.CountMemberGroupByRoleAndStatus(ctx)
+			for _, count := range countResult {
+				if count.Role == "OWNER" && count.RowStatus == "NORMAL" && count.Count == 1 {
```

Comment:
Does it always include the system robot? Should this be 2 instead of 1?

Reply:
It does now include the system bot, so 1 should be OK.